packages/plugins/rubygem-elastic-apm/rubygem-elastic-apm.spec

```diff
@@ -1,101 +1,82 @@
-# Generated from elastic-apm-2.9.1.gem by gem2rpm -*- rpm-spec -*-
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name elastic-apm
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.9.1
-Release: 3%{?dist}
+Name: rubygem-%{gem_name}
+Version: 4.7.3
+Release: 1%{?dist}
 Summary: The official Elastic APM agent for Ruby
-Group: Development/Languages
 License: Apache-2.0
 URL: https://github.com/elastic/apm-agent-ruby
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(concurrent-ruby) >= 1.0
-Requires: %{?scl_prefix}rubygem(concurrent-ruby) < 2
-Requires: %{?scl_prefix}rubygem(http) >= 3.0
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 2.3.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.3.0
+BuildRequires: ruby >= 2.3.0
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 %description
 The official Elastic APM agent for Ruby.
 
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+%setup -q -n  %{gem_name}-%{version}
 
 %build
 # Create the gem as gem install only works on a gem file
-%{?scl:scl enable %{scl} - << \EOF}
-gem build %{gem_name}.gemspec
-%{?scl:EOF}
+gem build ../%{gem_name}-%{version}.gemspec
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
-%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
-%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -pa .%{gem_dir}/* \
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %files
 %dir %{gem_instdir}
-%exclude %{gem_instdir}/.ci
+%{gem_instdir}/.ci
```

Comment:
```suggestion
%exclude %{gem_instdir}/.ci
```

Reply:
This is the result of `gem2rpm` with the default template.

Reply:
We can update that ;)

https://github.com/theforeman/foreman-packaging/blob/770f5380dde1ba4686db47ff2965a37c2570cd28/gem2rpm/default.spec.erb#L15-L42

Reply:
I've added the recommanded changes. This is probably because the `elastic-apm` gem isn't build by us and therefore doesn't comply with our expectations :wink: